librz/cons/cpipe.c

```diff
@@ -36,12 +31,8 @@ RZ_API int rz_cons_pipe_open(const char *file, int fdn, int append) {
 		eprintf("rz_cons_pipe_open: Cannot open file '%s'\n", file);
 		return -1;
 	}
-	if (backup_fd != -1) {
-		close(backup_fd);
-		// already set in __dupDescriptor // backup_fd = -1;
-	}
-	backup_fdn = fdn;
-	if (!__dupDescriptor(fd, fdn)) {
+	fds->backup_fdn = fdn;
+	if (!__dupDescriptor(fd, fdn, fds)) {
 		eprintf("Cannot dup stdout to %d\n", fdn);
```

Comment:
RZ_LOG_ERROR

Reply:
should i replace eprintf with RZ_LOG_ERROR?